components/brave_new_tab_ui/brave_new_tab.tsx

```diff
@@ -4,7 +4,7 @@
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import * as React from 'react'
-import { render } from 'react-dom'
+import {createRoot} from 'react-dom/client'
```

Comment:
Aren't we going with `{ createRoot }` style?

Reply:
Nice catch!